Cargo.toml

```diff
@@ -89,7 +87,7 @@ subst = { version = "0.3", features = ["yaml"] }
 testcontainers-modules = { version = "0.11.6", features = ["postgres"] }
 thiserror = "2"
 tiff = "0.9.1"
-tile-grid = "0.6"
+# tile-grid = "0.6"
```

Comment:
lets remove entirely instead of commenting out.

If intentional, a comment for future devs would be helpful

```suggestion
```

Reply:
oops, mised that, thx